src/platforms/posix/main.cpp

```diff
@@ -502,9 +502,11 @@ int main(int argc, char **argv)
 
 	if (!daemon_mode) {
 		string mystr;
+		string add_string;
 		string string_buffer[CMD_BUFF_SIZE];
 		int buf_ptr_write = 0;
 		int buf_ptr_read = 0;
+		uint8_t cursor_position = 0;
```

Comment:
Might be worth adding a note that it's counted from right to left (0 = end of string)